### PR TITLE
add Role resource

### PR DESCRIPTION
**Pull request by jaypipes**
Adds the Role resource. Note that the PermissionsBoundary field has a
different Go type between the Create input shape (`*string`) and the
Create and ReadOne output shapes (`AttachedPermissionsBoundary` struct)
and I generated this code using a not-yet-merged ack-generate that
supports nested field translations.

Also note the renames from the Role.RoleName to just Role.Name.

Updates the iam-controller's `go.mod` to bring in ACK runtime 0.15.2

By submitting this pull request, I confirm that my contribution is made under
the terms of the Apache 2.0 license.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/aws-controllers-k8s/iam-controller
 go 1.14
 
 require (
-	github.com/aws-controllers-k8s/runtime v0.14.1
+	github.com/aws-controllers-k8s/runtime v0.15.2
 	github.com/aws/aws-sdk-go v1.37.10
```
Comment (A-Hilaly) [inline go.mod:7]: Maybe generate with the most recent version or `v1.42`

File: test/e2e/tests/test_role.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+        latest = role.get(role_name)
+        assert latest is not None
+        assert latest['MaxSessionDuration'] == max_sess_duration
+
+        new_max_sess_duration = max_sess_duration + 100
+
+        # We're now going to modify the CopyTagsToSnapshot field of the DB
+        # instance, wait some time and verify that the RDS server-side resource
+        # shows the new value of the field.
```
Comment (A-Hilaly) [inline test/e2e/tests/test_role.py:82]: Copy pasta :p

**Review by A-Hilaly — APPROVED** [review]
Alles gut!

**Comment by A-Hilaly** [comment]
Sent https://github.com/aws-controllers-k8s/test-infra/pull/165 to enable prow jobs for this controller

File: test/e2e/tests/test_role.py (hunk 2 above)
Comment (jaypipes) [reply test/e2e/tests/test_role.py:82]: fixed, nice catch :)

File: go.mod (hunk 1 above)
Comment (jaypipes) [reply go.mod:7]: done.

**Review by vijtrip2 — APPROVED** [review]

**Comment by A-Hilaly** [comment]
Tests passed on my local machine. Merging

```
make kind-test SERVICE=iam 
checking AWS credentials ... ok.
creating kind cluster ack-test-36755d47-f7915f8c ... Creating cluster "ack-test-36755d47-f7915f8c" ...
 ✓ Ensuring node image (kindest/node:v1.16.9) 🖼
 ✓ Preparing nodes 📦 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
 ✓ Joining worker nodes 🚜 
Set kubectl context to "kind-ack-test-36755d47-f7915f8c"
You can now use your cluster with:

kubectl cluster-info --context kind-ack-test-36755d47-f7915f8c --kubeconfig /home/amine/source/github.com/aws-controllers-k8s/test-infra/scripts/../build/tmp-ack-test-36755d47-f7915f8c/kubeconfig

Have a nice day! 👋
ok.
building aws-controllers-k8s:iam-c8af98b-dirty docker image ... ok.
loading the images into the cluster ... ok.
================================================================================
test-helm.sh] Starting Helm Artifacts Test
test-helm.sh] installing the Helm Chart ack-iam-controller in namespace ack-system-test-helm ... ok.
test-helm.sh] ACK iam controller deployment name is ack-system-test-helm/ack-iam-controller-iam-chart
test-helm.sh] Generating AWS temporary credentials and adding to env vars map in controller deployment ... ok.
test-helm.sh] waiting 10 seconds for iam controller to start ... ok
test-helm.sh] ACK iam controller pod name is ack-system-test-helm/ack-iam-controller-iam-chart-d46c7d984-wtvrn
test-helm.sh] Verifying that pod status is in Running state ... ok.
test-helm.sh] Verifying that there are no ERROR in controller logs ... ok.
test-helm.sh] uninstalling the Helm Chart ack-iam-controller in namespace ack-system-test-helm ... ok.
test-helm.sh] removing iam crds installed by Helm ... ok.
test-helm.sh] deleting ack-system-test-helm namespace ... ok.
test-helm.sh] Helm Artifacts Test Finished Successfully
================================================================================
loading CRD manifests for iam into the cluster ... ok.
loading common manifests into the cluster ... ok.
loading RBAC manifests for iam into the cluster ... ok.
loading service controller Deployment for iam into the cluster ...ok.
generating AWS temporary credentials and adding to env vars map ... ok.
================================================================================
To poke around your test cluster manually:
export KUBECONFIG=/home/amine/source/github.com/aws-controllers-k8s/test-infra/scripts/../build/tmp-ack-test-36755d47-f7915f8c/kubeconfig
kubectl get pods -A
================================================================================
Building e2e test container for iam
Running e2e test container sha256:72768f2a1332e113f67ae207f1af7f5c2b75838ff2e85c08e74a0d30f429b654
running python tests locally...
INFO:root:🛠️ Bootstrapping resources ...
INFO:root:Wrote bootstrap to /iam-controller/tests/e2e/bootstrap.pkl
=================================================================== test session starts ===================================================================
platform linux -- Python 3.8.10, pytest-6.2.5, py-1.11.0, pluggy-1.0.0
rootdir: /iam-controller/tests/e2e
plugins: forked-1.3.0, xdist-2.2.0
gw0 I / gw1 I / gw2 I / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 I / gw2 I / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 I / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 I / gw4 I / gw5 I / gw6 I / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 I / gw5 I / gw6 I / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 I / gw6 I / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 I / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 I / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 I / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 I / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 I / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 I / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 I / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 I / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 I / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 I / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 I / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 I / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / gw1gw0 C / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / gw1[gw0] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                             
gw0 ok / gw1 C / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / gw[gw1] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                              
gw0 ok / gw1 ok / gw2 C / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / g[gw2] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                               
gw0 ok / gw1 ok / gw2 ok / gw3 C / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C / [gw3] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                
gw0 ok / gw1 ok / gw2 ok / gw3 ok / gw4 C / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C /[gw4] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                 
gw0 ok / gw1 ok / gw2 ok / gw3 ok / gw4 ok / gw5 C / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C [gw5] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                  
gw0 ok / gw1 ok / gw2 ok / gw3 ok / gw4 ok / gw5 ok / gw6 C / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 C[gw6] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                   
gw0 ok / gw1 ok / gw2 ok / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 C / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17 [gw7] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                    
gw0 ok / gw1 ok / gw2 ok / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 C / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw17[gw8] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                     
gw0 ok / gw1 ok / gw2 ok / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 C / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw1[gw9] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                      
gw0 ok / gw1 ok / gw2 ok / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gwgw0 [1] / gw1 ok / gw2 ok / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / ggw0 [1] / gw1 [1] / gw2 ok / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C / gw0 [1] / gw1 [1] / gw2 [1] / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 C / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C /[gw10] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                         
gw0 [1] / gw1 [1] / gw2 [1] / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 C / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C [gw11] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                          
gw0 [1] / gw1 [1] / gw2 [1] / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 C / gw13 C / gw14 C / gw15 C / gw16 C[gw12] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                           
gw0 [1] / gw1 [1] / gw2 [1] / gw3 ok / gw4 ok / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 C / gw14 C / gw15 C / gw16 gw0 [1] / gw1 [1] / gw2 [1] / gw3 ok / gw4 [1] / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 C / gw14 C / gw15 C / gw16gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 C / gw14 C / gw15 C / gw1[gw13] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                              
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 ok / gw6 ok / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 C / gw15 C / gwgw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 ok / gw6 [1] / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 C / gw15 C / g[gw14] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 ok / gw6 [1] / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 C / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 C /[gw15] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                  
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 ok [gw16] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                   
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 ok / gw8 ok / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 ok gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 ok / gw8 [1] / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 okgw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 o[gw17] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                      
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 o[gw18] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                       
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 ok / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 ogw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 ok / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15 gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 ok / gw12 ok / gw13 ok / gw14 ok / gw15gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 ok / gw12 [1] / gw13 ok / gw14 ok / gw1[gw19] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                           
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 ok / gw12 [1] / gw13 ok / gw14 ok / gw1gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 ok / gw14 ok / gw[gw21] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                             
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 ok / gw14 ok / gwgw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 ok / g[gw20] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                               
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 ok / g[gw22] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                                
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 ok / ggw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 ok / ggw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / [gw23] Python 3.8.10 (default, May  4 2021, 19:07:02)  -- [GCC 10.2.1 20201203]                                                                                                                                                                   
gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw0 [1] / gw1 [1] / gw2 [1] / gw3 [1] / gw4 [1] / gw5 [1] / gw6 [1] / gw7 [1] / gw8 [1] / gw9 [1] / gw10 [1] / gw11 [1] / gw12 [1] / gw13 [1] / gw14 [1] / gw15 [1] / gw16 [1] / gw17 [1] / gw18 [1] / gw19 [1] / gw20 [1] / gw21 [1] / gw22 [1] / gw23 [1]
scheduling tests via LoadScheduling

tests/test_role.py::TestRole::test_crud 
[gw0] [100%] PASSED tests/test_role.py::TestRole::test_crud 

============================================================== 1 passed in 130.46s (0:02:10) ==============================================================
INFO:root:🧹 Cleaning up resources ...
To resume test with the same cluster use: " TMP_DIR=/home/amine/source/github.com/aws-controllers-k8s/test-infra/scripts/../build/tmp-ack-test-36755d47-f7915f8c
    AWS_SERVICE_DOCKER_IMG=aws-controllers-k8s:iam-c8af98b-dirty "
```

**Pull request merged**